.buildkite/pipeline.yml

```diff
@@ -431,9 +429,42 @@ steps:
           slurm_mem: 20GB
           slurm_gpus: 1
 
-
-
-      - wait
+  # - group: "Bash scripts"
+
+  #   steps:
+
+  #     - label: "Submit and Monitor sbatch Job on Caltech HPC"
+  #       command: |
+  #         #!/bin/bash
+
+  #         # Submit the sbatch script and capture its job ID
+  #         JOB_ID=$(sbatch  test/mpi_tests/local_checks.sh | awk '{print $4}')
+  #         echo "Submitted job with ID: $JOB_ID, output log: slurm-$JOB_ID.out"
+  #         START_TIME=$(date +%s)
+
+  #         # Loop until the job finishes
+  #         while true; do
+  #             # Check the status of the job
+  #             STATUS=$(squeue -j $JOB_ID | grep $JOB_ID | awk '{print $5}')
+  #             ELAPSED_TIME=$(( $(date +%s) - $START_TIME ))
+
+  #             # If the job status is 'PD' (pending) or 'R' (running), wait and continue checking
+  #             if [ "$STATUS" == "PD" ] || [ "$STATUS" == "R" ]; then
+  #                 sleep 60
+  #                 echo "Job is still running... Elapsed time: $ELAPSED_TIME seconds."
+  #             # If the job status is 'CF' (completed successfully), print success message and exit
+  #             elif [ "$STATUS" == "CF" ]; then
+  #                 echo "Job completed successfully."
+  #                 exit 0
+  #             # If the job status is anything else, print error message and exit
+  #             else
+  #                 echo "Error: Job failed or terminated."
```

Comment:
Maybe we can also print out the stdout/stderr for the job so that users can see the issue in addition to just seeting the return code.